### DISCUSSION TITLE
2.0.1 is out!

### DISCUSSION THREAD

**Discussion by Gissio** [Gissio/radpro#109] [Announcements]
Rad Pro 2.0.1 is out, featuring minor enhancements.

It is now possible to reset the data log.

As there are several versions of the J613/J614 tubes, with the most common conversion factor being 60 cpm/µSv/h, the default conversion factor has been changed to 60 cpm/µSv/h.

Also, some issues involving custom HV profiles and data communications have been fixed.

Enjoy!

**Comment by alfmck** [comment]
Tried to accumulate datalog for 60 hours (full memory) and save new history data. Process succeeded but after that new history data saving was unavailable. Only after logging reset and saving all history data, I was able to read new history data again, but number of records sometimes is unrealistic.
